graphjet-core/src/main/java/com/twitter/graphjet/algorithms/counting/TopSecondDegreeByCount.java

```diff
@@ -74,14 +74,23 @@ public TopSecondDegreeByCount(
   }
 
   /**
+   * Returns whether we should update the node info, according the request requirements.
```

Comment:
no need to mention node info here, as it is not used in this function.
Just say
Return whether the edge is within the age limit which is specified in the request.